client/examples/docker/Dockerfile

```diff
@@ -4,9 +4,12 @@ FROM openjdk:8-jdk
 
 USER root
 
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+RUN curl -sL  https://deb.nodesource.com/setup_8.x | bash -
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
 
-RUN apt-get install -y nodejs
+#RUN apt-get update
+RUN apt-get install -y nodejs yarn
```

Comment:
This native installation of yarn does not work for me :-/

It works if I install the yarn package globally via npm, though that way is now deprecated.

Reply:
I found out the problem is the missing `apt-get update`. It is actually mandatory, otherwise the previously configured package repo is not considered.